src/Microsoft.Health.Fhir.Shared.Core/Extensions/ModelExtensions.cs

```diff
@@ -20,11 +20,7 @@ public static class ModelExtensions
         /// </summary>
         public static void SetModelInfoProvider()
         {
-#if Stu3
-            ModelInfoProvider.SetProvider(new Stu3ModelInfoProvider());
-#elif R4
-            ModelInfoProvider.SetProvider(new R4ModelInfoProvider());
-#endif
+            ModelInfoProvider.SetProvider(new VersionSpecificModelInfoProvider());
```

Comment:
Can we remove the macro definition now?

Reply:
There is still 1 more use in `SearchParameterDefinitionBuilder`